src/vm/customattribute.cpp

```diff
@@ -294,13 +294,11 @@ HRESULT Attribute::ParseCaValue(
         IfFailGo(ca.GetU4(&len));
         pCaArg->arr.length = len;
         pCaArg->arr.pSArray = NULL;
-        if (pCaArg->arr.length == (ULONG)-1)
-            break;
 
         IfNullGo(pCaArg->arr.pSArray = pCaValueArrayFactory->Create()); 
         elementType.Init(pCaArg->type.arrayType, SERIALIZATION_TYPE_UNDEFINED, 
```

Comment:
Oh, now I understand your point... clear, let me double check, I will add another commit for this...